analyses/oncoplots/02-prepare_files_oncogrid.R

```diff
@@ -103,8 +103,9 @@ annot_info <- hist_df %>%
                 "Diagnosis_Type" = "HOPE_diagnosis_type",
                 "Tumor_Location" = "HOPE_Tumor.Location.condensed",
                 "Sex" = "HARMONY_Gender",
-                "Age" = "HARMONY_age_class_derived") %>%
-  dplyr::select(Sample, Sequencing_Experiment, Diagnosis, Diagnosis_Type, Tumor_Location, Sex, Age, TMB) %>%
+                "Age" = "HARMONY_age_class_derived",
+                "Molecular_Subtype" = "molecular_subtype") %>%
+  dplyr::select(Sample, Sequencing_Experiment, Diagnosis, Molecular_Subtype, Diagnosis_Type, Tumor_Location, Sex, Age, TMB) %>%
```

Comment:
hi @komalsrathi line 104, can you make that `CNS_region` instead?  Also trying to get v2 out today so we will update instead of `molecular_subtype` using `cancer_group_short`... will let you know when that's done.

Reply:
cc @mkoptyra

Reply:
Just saw this - somehow missed it last week. 

1. So for the oncoplots, we don't need `molecular_subtype` in the annotation, but instead we want `cancer_group_short`? I also have `Diagnosis` (which is a combination of `HOPE_diagnosis` + `HARMONY_WHO.Grade`) in the annotation, should I keep it or remove it? 

2. Also, should `Tumor_Location` be replaced with `CNS_region`?